python-main.js

```diff
@@ -934,7 +934,38 @@ function web_editor(config) {
             modalMsg(config['translate']['load']['invalid-file-title'], config['translate']['load']['extension-warning'],"");
         }
     }
-
+    // Trap focus in modal and pass focus to first actionable element
+    function focusModal() {
+        document.querySelector('body > :not(.vex)').setAttribute('aria-hidden', true)
+        var dialog = document.querySelector('.vex-content', '.modal-overlay');
+        var focusableEls = dialog.querySelectorAll('a:not([disabled]), a[href]:not([disabled]), button:not([disabled]), textarea:not([disabled]), input[type="text"]:not([disabled]), input[type="radio"]:not([disabled]), input[type="checkbox"]:not([disabled]), select:not([disabled]),td:not([disabled])');
+        $(focusableEls).each(function(){
+            $(this).attr('tabindex', '0'); 
+        });
+        focusableEls[0].focus();
+        dialog.onkeydown = function(event) {
+        if (event.which == 9) {
```

Comment:
What does this do? I assume 9 represent a specific key? It's be good to have a comment here.